src/project_types/permissions.rs

```diff
@@ -91,4 +78,25 @@ impl Permission {
             Err(Error::InvalidPermissionTarget)
         }
     }
+
+    /// Does the grunt work of transforming a sequence section target
+    /// of the form seq_name,sec_num into a tuple of those values,
+    /// with sec_num casted
+    pub fn parse_seq_sec_target(
+        target: &Option<String>
+    ) -> Result<(String, u32), Error> {
+        let target_str = target.to_owned().expect("Error unwrapping target");
```

Comment:
Also, this should be a match, not an expect. Because it was an optional parameter, docopt hasn't confirmed it's there, and we don't want a panic.